content/courses/advanced-ada/chapters/numerics.rst

```diff
@@ -1639,12 +1639,762 @@ precision, 0.2774 is rounded to 0.277, and 0.2777 is rounded to 0.278.
 Big Numbers
 -----------
 
+As we've seen before, we can define numeric types in Ada with a high degree of
+precision. However, these normal numeric types in Ada are limited to what
+the underlying hardware actually supports. For example, any signed integer
+type |mdash| whether defined by the language or the user |mdash| cannot have a
+range greater than that of :ada:`System.Min_Int .. System.Max_Int` because
+those constants reflect the actual hardware's signed integer types. In certain
+applications, that precision might not be enough, so we have to rely on
+:wikipedia:`arbitrary-precision_arithmetic <arbitrary-precision_arithmetic>`.
+Ada's *big* numbers aren't constrained by the underlying hardware, allowing
+us to operate with this kind of arithmetic.
```

Comment:
Actually, given the text in the next paragraph, I suggest changing the entire last sentence to the following:

These so-called "big numbers" are limited conceptually only by available memory, in contrast to the underlying hardware-defined numeric types.

Reply:
Perfect, thanks!